pkgs/development/ocaml-modules/extlib/1.7.8.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, lib, fetchurl, fetchpatch, ocaml, findlib, cppo, minimal ? true }:
+
+assert lib.versionAtLeast (lib.getVersion ocaml) "3.11";
+
+stdenv.mkDerivation rec {
+  pname = "ocaml${ocaml.version}-extlib";
+  version = "1.7.8";
+
+ src = fetchurl {
+    url = "https://ygrek.org/p/release/ocaml-extlib/extlib-${version}.tar.gz";
+    sha256 = "0npq4hq3zym8nmlyji7l5cqk6drx2rkcx73d60rxqh5g8dla8p4k";
+  };
+
+  buildInputs = [ ocaml findlib cppo ];
+
+  createFindlibDestdir = true;
+
+  dontConfigure = true;      # Skip configure
+  # De facto, option minimal=1 seems to be the default.  See the README.
+  buildPhase     = "make ${if minimal then "minimal=1" else ""} build";
+  installPhase   = "make ${if minimal then "minimal=1" else ""} install";
```

Comment:
```suggestion
  buildPhase     = "make ${minimal_flag} build";
  installPhase   = "make ${minimal_flag} install";
```

Reply:
Probably not worth it refactoring this as 1.7.8 is only meant as a temporary thing in nixpkgs to update haxe. I promise I'll refactor the normal extlib expression when 1.7.9 comes around and this can be finally deleted :)